src/pmx/analysis.py

```diff
@@ -94,7 +94,7 @@ def read_dgdl_files(lst, lambda0=0, invert_values=False, verbose=True,\
             except:
                 print(' !! Error in checking %s' % (lst[idx]))
                 good=False
-                idx+=1
+        idx+=1
```

Comment:
This will increment the index every time and the next iteration will start by skipping one dhdl. I suggest to leave line 97 as was previously, but add lines 98-99:
```
        else:
            idx+=1
```

Reply:
Oops, thanks for catching that!